src/server/tran_server.cpp

```diff
@@ -217,7 +217,9 @@ tran_server::init_page_server_hosts (const char *db_name)
     {
       //at least one valid host exists clear the error remaining from previous failing ones
       er_clear ();
+      exit_code = NO_ERROR;
```

Comment:
@hornetmj Even if `local_exit_cde` is removed, the `exit_code` is needed and cleared when the last connections fails. 

I assume that use `exit_code` instead of `local_exit_code` used for `connect_to_page_server`. Please tell me know more details of suggestion if it's not you meant.

Reply:
@wrlawodms I think the local varable local_exit_code is useless. So, I gave you a suggestion.

Reply:
@hornetmj Oh, I got it finally. :) and it's done.